tests/cpp_unit_tests/test_tap_position_optimizer.cpp

```diff
@@ -290,7 +287,7 @@ TEST_CASE("Test Transformer ranking") {
             }
 
             pgm_tap::TrafoGraphEdgeProperties const regulated_edge_weights = get_edge_weights(g);
-            pgm_tap::TrafoGraphEdgeProperties const ref_regulated_edge_weights{{{0, 1}, 0}, {{2, 3}, 2}};
```

Comment:
Shouldnt thishave been {{{0, 1}, 0}, {{2, 3}, 3}} before?

Reply:
The internal logic now is very sensitive to the values we assign. Before we could assign arbitrary non-negative weight, whereas since the restrictive check, we can no longer do that. The changed weights in this test are reflected as a result of changing edge properties in the same context.

Reply:
Resolved per discussion 'offline'